pages/deposit/[symbol].tsx

```diff
@@ -101,14 +100,13 @@ const Deposit = () => {
                 epochs={epochs}
                 setEpochs={setEpochs}
                 actionButtonProps={{
-                  disabled:
-                    amount === 0n || epochs === 0 || amount > maxDepositAmount,
+                  disabled: amount === 0n || amount > maxDepositAmount,
                   onClick: async () => {
                     const { timestamp } = await publicClient.getBlock()
                     const hash = await deposit(
                       asset,
                       amount,
-                      epochs,
+                      epochs + 1,
```

Comment:
```suggestion
                      epochs,
```
@Dorvin Let's move +1 logic to deposit context